supportedBackends/tes/src/test/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActorSpec.scala

```diff
@@ -70,6 +82,15 @@ class TesAsyncBackendJobExecutionActorSpec
     content = None
```

Comment:
These tests are great - nice mocking! I would love to see more, though, particularly around the polling (and then not polling) behavior, and how we respond to various forms of the FULL_VIEW from TES. You can set up Actor-level testing that inspects the metadata stored by actions, which would be really useful here.